README.md

```diff
@@ -1,20 +1,134 @@
+[releases]: https://github.com/aeternity/aeternity/releases
```

Comment:
These references usually stay at the bottom of the file

Reply:
44f4c4d